### PR TITLE
Add persona meta text

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | none |
| Versions      | all |
| Edition       | all |

Add meta text to main persona path articles, just in case they end up as being parts of a larger master page.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date

### CODE REVIEW

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/persona_paths/shop_manager.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+description: Perform tasks related to running a virtual store.
```
Comment (dabrt) [inline docs/persona_paths/shop_manager.md:2]:
```suggestion
description: Perform tasks related to running an electronic presence of your store.
```

[the remark this answers is not in the document]
Comment (dabrt) [reply docs/persona_paths/shop_manager.md]: Fixed

File: docs/persona_paths/administrator.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+---
+description: Discover administrative tasks that you can perform in the back Office.
```
Comment (juskora) [inline docs/persona_paths/administrator.md:2]:
```suggestion
description: Discover administrative tasks that you can perform in the Back Office.
```

File: docs/persona_paths/shop_manager.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+---
+description: Perform tasks related to running an electronic presence of your store.
```
Comment (juskora) [inline docs/persona_paths/shop_manager.md:2]: How about sth with "online store"?

File: docs/persona_paths/shop_manager.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
+---
+
 # Shop manager role
```
Comment (juskora) [inline docs/persona_paths/shop_manager.md:5]: It shouldn't be Store manager according to the discussion about the landing page tiles in the user-doc?

File: docs/persona_paths/shop_manager.md (hunk 3 above)
Comment (juskora) [reply docs/persona_paths/shop_manager.md:2]: This must be changed. Electronic presence is not a proper wording here.

**Review by juskora — CHANGES_REQUESTED** [review]

**Review by juskora — APPROVED** [review]

File: docs/persona_paths/organize_content.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Organize the content of your website by copyingor moving Content items, controlling Locations and URLs and applying classification techniques.
+description: Organize the content of your website by copying or moving Content items, controlling Locations and URLs and classifying content.
```
Comment (adriendupuis) [inline docs/persona_paths/organize_content.md:2]:
```suggestion
description: Organize the content of your website by copying or moving Content items, controlling Locations and URLs, and classifying content.
```

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**